File: usr/source/fort/rt1/clog.f
	function clog(z1)
	complex clog, z1, z2
	dimension dummy(2)
	equivalence (a,z2,dummy(1)), (b,dummy(2))

	z2 = z1
	c = cabs(z2)
	b = atan2(b, a)
	a = alog(c)
	clog = z2
	return
	end

File: .ref-c1c75ba2110baece8edca003eb2d4fe2511b0e4b/usr/src/usr.bin/f77/libF77/c_log.c
/*
 * Copyright (c) 1980 Regents of the University of California.
 * All rights reserved.  The Berkeley software License Agreement
 * specifies the terms and conditions for redistribution.
 *
 *	@(#)c_log.c	5.1	%G%
 */

#include "complex"

c_log(r, z)
complex *r, *z;
{
double log(), cabs(), atan2();

r->imag = atan2(z->imag, z->real);
r->real = log( cabs(z->real, z->imag) );
}
